lib/site-inspector.rb

```diff
@@ -49,8 +51,13 @@ def domain
     www? ? PublicSuffix.parse("www.#{@uri.host}") : @domain
   end
 
-  def request(ssl=false, www=false, followlocation=true)
-    Typhoeus.get(uri(ssl, www), followlocation: followlocation, timeout: 10)
+  def request(ssl=false, www=false, followlocation=true, ssl_verifypeer=true, ssl_verifyhost=true)
```

Comment:
This is getting ugly. Can we make this an options hash with defaults (which is a bit more Ruby)?